monai/bundle/scripts.py

```diff
@@ -185,6 +189,9 @@ def download(
 
     Args:
         name: bundle name. If `None` and `url` is `None`, it must be provided in `args_file`.
+            for example: "spleen_ct_segmentation", "prostate_mri_anatomy" in the model-zoo:
+            https://github.com/Project-MONAI/model-zoo/releases/tag/hosting_storage_v1.
+        version: version name of the target bundle to download, like: "0.1.0" or "v0.1.0".
```

Comment:
v0.1.0 starting with `v` is not a proper version number, I don't think we should support it throughout this project and in the model zoo. https://semver.org/#is-v123-a-semantic-version

Reply:
OK, that would simplify the logic.
Updated in the latest commit.

Thanks.